<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,0,0,0,5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>Server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="serverLn">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>10.0.201.12:1666</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>Workspace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="workLn">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>qinjiaxin_01YXHY1235_Assets</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Microsoft JhengHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="userLn">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>qinjiaxin</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5,0">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordLn">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="passwordBtn">
            <property name="minimumSize">
             <size>
              <width>12</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	min-width:10px;
    background-color: rgba(39, 44, 54, 0);
	background-position: center;
	background-repeat: no-repeat;
	background-image: url(:/icons/icons/no_display_password.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="connectBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>110</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">/* /////////////////////////////////////////////////////////////////////////////////////////////////
Custom */
#connectBtn{ font-family: &quot;Microsoft YaHei&quot;;
    font-size: 15px;
	font-color:#2b2b2b;
border-radius: 20px;
background-color: #214283;
background-repeat: no-repeat;
background-position: left center;
}
#connectBtn:hover { background-color: #282c34; border-style: solid; border-radius: 20px; }
#connectBtn:pressed { background-color: #214283; border-style: solid; border-radius: 20px; }</string>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="currentPathCombox">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Asset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="assetNameComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Step</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="submitStepCom">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>368</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>AssetName</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="workTree">
          <property name="styleSheet">
           <string notr="true">QTreeWidget::branch:closed:has-children{
    image:url(icons/close_folder.png);
}

QTreeWidget::branch::open::has-children{
    image:url(icons/open_folder.png);
}
</string>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Submit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>单文件提交</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QSplitter" name="splitter_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>File List</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListWidget" name="listWidget"/>
                </item>
               </layout>
              </widget>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Comment</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTextEdit" name="textEdit"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="publishBtn">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Publish</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>多文件提交</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Under functional development...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>passwordBtn</sender>
   <signal>pressed()</signal>
   <receiver>passwordLn</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordBtn</sender>
   <signal>pressed()</signal>
   <receiver>passwordBtn</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
